### PR TITLE
BrenKt-SK-1331-FR-WEB-Exchange-configuring_email_for_Android

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
What's new ?
- new images
- deleting old images
- Fr update

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md [hunk 1]
```diff
@@ -1,82 +1,106 @@
...
+> **Etape 2**
+>> Saisissez votre adresse e-mail puis appuyez sur `Suivant`{.action}.<br><br>
+>> ![exchange android](images/exchange-android-02.png){.thumbnail .h-600}
+>>
+> **Etape 4**
+>> Saisissez le mot de passe de votre adresse e-mail puis appuyez sur `Suivant`{.action}.<br><br>
+>> ![exchange android](images/exchange-android-03.png){.thumbnail .h-600}
+>>
+> **Etape 5**
+>> Complétez la page « **Configuration de l'adresse** »<br><br>- **E-mail** : Votre adresse e-mail complète<br>- **Mot de passe** : Le mot de passe de votre adresse e-mail<br>- **Certificat** : laissez « Aucun »<br>- **Domaine\Nom d'utilisateur** : Votre adresse e-mail complète<br>- **Serveur** : le nom de domaineattaché à votre compte e-mail<br>- **Port** : « 443 »<br>- **Type de sécurité** : « SSL/TLS »<br><br>Appuyez sur `Suivant`{.action} pour valider la configuration.<br><br
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md]:
```suggestion
>> Complétez la page « **Configuration de l'adresse** »<br><br>- **E-mail** : votre adresse e-mail complète<br>- **Mot de passe** : le mot de passe de votre adresse e-mail<br>- **Certificat** : laissez « Aucun »<br>- **Domaine\Nom d'utilisateur** : votre adresse e-mail complète<br>- **Serveur** : le nom de domaine attaché à votre compte e-mail<br>- **Port** : 443<br>- **Type de sécurité** : SSL/TLS<br><br>Appuyez sur `Suivant`{.action} pour valider la configuration.<br><br
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md [hunk 2]
```diff
@@ -1,82 +1,106 @@
...
+>> ![exchange android](images/exchange-android-04.png){.thumbnail .h-600}
+>>
+> **Etape 6**
+>> Le message « Vous pouvez dés à présent utiliser votre compte » devrait apparaitre, appuyez sur `OK`{.action} pour terminer la configuration.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md]:
```suggestion
>> Le message « Vous pouvez dès à présent utiliser votre compte » devrait apparaitre, appuyez sur `OK`{.action} pour terminer la configuration.<br><br>
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md [hunk 3]
```diff
@@ -1,82 +1,106 @@
 ---
-title: Configurer son compte Exchange sur Android via l'application Gmail
-excerpt: Apprenez à configurer un compte Exchange sur Android, via l'application Gmail
-updated: 2018-03-12
+title: "Exchange - Configurer une adresse e-mail dans Gmail pour Android"
+excerpt: "Apprenez à configurer un compte Exchange sur Android, via l'application Gmail"
+updated: 2024-03-19
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_android/guide.fr-fr.md]:
```suggestion
updated: 2024-03-20
```

**Pull request merged**